libs/cosmos-sdk/store/rootmulti/rootmulti_store.go

```diff
@@ -364,7 +369,16 @@ func (rs *Store) checkAndResetPruningHeights(roots map[int64][]byte) error {
 func (rs *Store) getCommitID(infos map[string]storeInfo, name string) types.CommitID {
 	info, ok := infos[name]
 	if !ok {
-		return types.CommitID{Version: tmtypes.GetStartBlockHeight()}
+		//for ibc upgrade repair version lower than Venus1Height
+		height := tmtypes.GetVenus1Height()
+		startVersion := viper.GetInt64(FlagStartHeight)
```

Comment:
you need cache viper.GetInt64 because it will cost performance

Reply:
fixed